js/src/edit.js

```diff
@@ -35,6 +37,10 @@ function registerStoreInGutenberg() {
 	} );
 }
 
+const YoastSidebarContainer = styled.div`
```

Comment:
It seems to me this padding shouldn't be on the sidebar itself, but on the underlying components. For instance, a `<Collapsible>` should have the padding (which it already does).